packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/workflow-config.ts

```diff
@@ -0,0 +1,14 @@
+import { TokenSymbol } from '@cardstack/web-client/utils/token';
+import BN from 'web3-core/node_modules/@types/bn.js';
+
+export const faceValueOptions = [5000, 10000, 50000, 100000];
+export const spendToUsdRate = 0.01;
+
+export interface Token {
+  address?: string;
+  balance?: BN;
+  icon: string;
+  name: string;
+  description?: string;
+  symbol: TokenSymbol;
+}
```

Comment:
Not sure where this should go

Reply:
Do we need this type, if we store the token's symbol and balance separately?

Reply:
I liked the convenience of having the address and balance be readily available. But now I'm wondering if there's the risk of not having the balances be updated. I think I'll give up on the convenience.